src/pages/ApplyMain/index.js

```diff
@@ -15,14 +15,27 @@ const ApplyMainPage = () => {
   const [innerWidth, setInnerWidth] = useState(window.innerWidth);
   const DESKTOP_WIDTH = 1199;
   const TABLET_WIDTH = 599;
+  const [clikedArray, setClickedArray] = useState([false, false, false]);
+
+  const handleClick = idx => {
+    let newArr = clikedArray;
```

Comment:
이러면 shallow copy 가 되어버려서 clickedArray 도 수정되어 버립니다.
그런데 React state 는 그 특성상 수정되면 안되잖아요? 따라서 아래처럼 바뀌면 됩니다

```suggestion
    let newArr = [...clickedArray];
```